### ISSUE TITLE
Type not found - mphx...

### ISSUE THREAD

**Issue by Jarrio** [5Mixer/mphx#38]
Hello,

I'm not sure if I'm doing something wrong or not but after referencing mphx in the project.xml file using:
`<haxelib name="mphx"/>` and then either trying to import or reference mphx in any way results in the compiler not recognizing the library at all?

I installed via cmd using "haxelib install mphx" and I was trying to follow the examples but even doing something like "mphx.connection.IConnection" or "mphx.client.Client" results in HaxeDevelop saying: "Type not found: X"

Has any code changed or did I install this incorrectly?

**Comment by 5Mixer** [comment]
Try haxelib git mphx https://github.com/5Mixer/mphx.git

I think that the Haxelib version is out of date. If the problem persists I'll take a look tonight, and update the haxelib version.

On 3 Aug 2016, at 8:22 am, Jazz914 <notifications@github.com<mailto:notifications@github.com>> wrote:

Hello,

I'm not sure if I'm doing something wrong or not but after referencing mphx in the project.xml file using:
<haxelib name="mphx"/> and then either trying to import or reference mphx in any way results in the compiler not recognizing the library at all?

I installed via cmd using "haxelib install mphx" and I was trying to follow the examples but even doing something like "mphx.connection.IConnection" or "mphx.client.Client" results in HaxeDevelop saying: "Type not found: X"

Has any code changed or did I install this incorrectly?

**Comment by Jarrio** [comment]
Thanks that managed to fix the issue I was having, I have another question. When adding "new Main();" to the static function main(); I get the following error:

Server.hx:74: Server active on 127.0.0.1:8000. Code after server.start() will not run.
Called from sys.net.Socket::$statics line 1
Called from Main::main line 41
Called from Main::$init line 17
Called from Main::new line 32
Called from mphx.server.impl.Server::start line 78
Called from mphx.server.impl.Server::update line 97
Called from sys.net.Socket::select line 241
Called from sys.net.Socket::select line 235
Uncaught exception - Invalid call
Press any key to continue . . .

Which is unusual because I'm following your example and this is exactly what you've put in the "movement" example - which means the server kinda just runs and ends because it's not looping

Edit: Nevermind - I found the other post stating it's an issue with haxe, I installed a developers version and it works fine! Thanks for your help!
